src/base/bittorrent/torrentcreatorthread.cpp

```diff
@@ -160,3 +160,13 @@ void TorrentCreatorThread::run()
         emit creationFailure(QString::fromStdString(e.what()));
     }
 }
+
+int TorrentCreatorThread::calculatePiecesCount(const QString &inputPath, const int pieceSize)
+{
+    if (inputPath.isEmpty())
+        return 0;
+
+    libt::file_storage fs;
+    libt::add_files(fs, inputPath.toStdString(), fileFilter);
```

Comment:
OK, added back.
BTW, libtorrent seems to handle path separators perfectly...

>Also you can make button to be disabled after calculation and enabled again when path or piece size changed.

No, user might modify things after selecting path and need to recalculate it.